Edge Detection/Readme.md

```diff
@@ -0,0 +1,20 @@
+## **Edge Detection of a image**
+
+This script uses `OpenCV` for taking input and output for the image. In this we are using he `Canny algorithm` for the detection of the edge 
+## How to use it
+
+### You can use this in two ways
+* If your machine has any python intrepretor than you can go for [**Edge_Detection.py**](https://github.com/ShubhamGupta577/Amazing-Python-Scripts/blob/Edge-Detection-Using-OpenCV/Edge%20Detection/Edge_Detection.py)
+  * For these you need to install **OpenCV** in your machine.
+* Another way is to use the [**Google Colab file**](https://github.com/ShubhamGupta577/Amazing-Python-Scripts/blob/Edge-Detection-Using-OpenCV/Edge%20Detection/Edge_Detection.ipynb) which will be excecute on a cloud.
+
+## Technologies used
+* **Python**
+* **OpenCV**
+* **Google Colab**
+## Sample Input/Output
+<img src ="https://github.com/ShubhamGupta577/Testing/blob/main/Sample_Input.jpg" width=400/> <img src ="https://github.com/ShubhamGupta577/Testing/blob/main/Sample_Output.png" width=400/>
```

Comment:
Images still not rendered 
![image](https://user-images.githubusercontent.com/43691873/111575118-bfc2dc80-87d3-11eb-80fe-fe25687ca58f.png)

Reply:
The image links are broken